colcon_notification/event_handler/status.py

```diff
@@ -114,12 +114,12 @@ def __call__(self, event):  # noqa: D102
         elif isinstance(data, JobEnded):
             job = event[1]
             self._ended[job] = self._running[job]
-            self._ended[job]['end_time'] = time.time()
+            self._ended[job]['end_time'] = time.monotonic()
```

Comment:
It looks like the `end_time` key isn't used; consider removing it.  On the other hand, it has nice symmetry with the `start_time` key in the `_running` dictionary, so I don't feel strongly about it.